.travis.yml

```diff
@@ -1,4 +1,25 @@
 language: java
 
 jdk:
-  - oraclejdk8
\ No newline at end of file
+  - oraclejdk8
+
+services:
+  - postgres
+
+before_install:
+  - cd ~
+  - git clone https://github.com/ACWI-SSWD/nldi-db.git
+  - mkdir ~/liquibase
+  - cd ~/liquibase
+  - wget https://github.com/liquibase/liquibase/releases/download/liquibase-parent-3.4.2/liquibase-3.4.2-bin.tar.gz
+  - tar -xzf liquibase-3.4.2-bin.tar.gz -C ./
+  - cd ~/liquibase/lib
+  - wget https://jdbc.postgresql.org/download/postgresql-9.4-1204.jdbc42.jar
+
+before_script:
+  - psql -U postgres -c 'create database nldi;'
+  - psql -U postgres -c "create extension postgis"
```

Comment:
Is this getting done on the nldi database? This command looks different from the others too.